drivers/LVHDSR.py

```diff
@@ -36,7 +36,8 @@
 import cleanup
 import blktap2
 from journaler import Journaler
-from lock import Lock
+from lock import Lock, LOCK_TYPE_GC_RUNNING
+import cleanup
```

Comment:
Surely it must already have had cleanup imported as it was calling `cleanup.start_gc`? And the `LOCK_TYPE_GC_RUNNING` is no longer required

Reply:
In fact, yes, on line 36